### PR TITLE
chore: update example to RN0.61.5

### PR DESCRIPTION
**Pull request by Naturalclar**
# Summary

There seemed to be many people who are having trouble with using this library in RN 0.61.x, as shown in #43 and #50 

I've upgraded the example directory to get it working with RN 0.61.5

## Test Plan

| iOS | iOS |
| --- | --- |
|  |  |

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ❌     |

## Checklist

- [x] I have tested this on a device and a simulator
- [x] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: example/ios/example/AppDelegate.m [hunk 1]
```diff
@@ -34,31 +30,13 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   return YES;
 }
 
-// Required to register for notifications
-- (void)application:(UIApplication *)application didRegisterUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings
```
Comment (andrewtremblay-pear) [inline example/ios/example/AppDelegate.m]: Aren't these methods still required as long as we continue to target iOS <10.0?

Also, maybe it is time to add the more recent, non-deprecated methods.

**Comment by ecreeth** [comment]
@Naturalclar some changes have been made in the core: [D19700061](https://github.com/facebook/react-native/commit/edfdafc7a14e88a2660b95cb220c62f29b1b28c0)

Original summary:  
Resolve and reject promise for PushNotificationIOS.requestPermissions by **Paige Sun**

**Comment by Naturalclar** [comment]
@ecreeth Thanks for letting me know!
I'll look into it

**Comment by Entryist** [comment]
`import RCTDeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter';`

Shorter version works for me:

`import { DeviceEventEmitter } from 'react-native'`

**Comment by Naturalclar** [comment]
@Entryist Thanks so much for the info :+1: updated the example to use `DeviceEventEmitter`

**Pull request merged**